xml5ever/src/driver.rs

```diff
@@ -63,7 +63,8 @@ impl<Sink: TreeSink> TendrilSink<tendril::fmt::UTF8> for XmlParser<Sink> {
 
     fn process(&mut self, t: StrTendril) {
         self.input_buffer.push_back(t);
-        self.tokenizer.feed(&self.input_buffer);
+        // FIXME: Properly support </script> somehow.
+        let _ = self.tokenizer.feed(&self.input_buffer);
```

Comment:
This is okay, because it is exactly what `html5ever` does too: https://github.com/servo/html5ever/blob/31a2c319c4b9fb763c88fbb7b826e66c3ff372a0/html5ever/src/driver.rs#L107-L109